akka-samples/akka-sample-osgi-dining-hakkers/pom.xml

```diff
@@ -184,6 +180,14 @@
             <id>oss-sonatype-releases</id>
             <url>https://oss.sonatype.org/content/repositories/releases</url>
         </repository>
+        <repository>
+            <id>springsource-releases</id>
+            <url>http://repository.springsource.com/maven/bundles/release</url>
+        </repository>
```

Comment:
Without it, it was not able to resolve dependencies of the updated Karaf. E.g. things like `"org.springframework" % "org.springframework.aop" % "3.2.3.RELEASE"`